db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/DataDictionaryImpl.java

```diff
@@ -6592,7 +6678,208 @@ public void upgradeFixSystemColumnDefinition(CatalogRowFactory rowFactory,
         updateColumnDescriptor(cd,td.getUUID(),columnName,columnNameColArray,tc);
     }
 
+	/**
+	 * Get the UUID for the specified system table.  Prior
+	 * to Plato, system tables did not have canonical UUIDs, so
+	 * we need to scan systables to get the UUID when we
+	 * are updating the core tables.
+	 *
+	 * @param tableName		Name of the table
+	 * @param schemaUUID	UUID of schema
+	 * @param tc			TransactionController to user
+	 *
+	 * @return UUID	The UUID of the core table.
+	 *
+	 * @exception StandardException		Thrown on failure
+	 */
+    private UUID getUUIDForCoreTable(String tableName,
+                                     String schemaUUID,
+                                     TransactionController tc)
+				throws StandardException
+    {
+            ConglomerateController	heapCC;
```

Comment:
yeah, this is an ongoing issue inherited from Derby.  I didn't try to change all of the tabs in the ported Derby code.  Since our whole codebase has mixed tabs and spaces maybe we can address it through one large Jira.